process-engine/bpmn-studio
# Test & apply sqlite update

## Changes

1. Update sqlite3 and the full-stack server to use sqlite3 v5.0.0
2. Remove electron-rebuild dependency
3. Fix atlas-engine start command for the docker image
4. Install sqlite3 into the docker image

## Issues

PR: #2044 

## How to test the changes

See that you don't have to rebuild the sqlite3 binaries to use it with electron (dev/ `npm run electron-start-dev`).
Build the electron macOS/Windows app and be sure that they also start the atlas-engine.
